spec/models/course_spec.rb

```diff
@@ -12,19 +12,12 @@
   end
 
   it "has many users" do
+    course = FactoryGirl.build(:course)
+    john = FactoryGirl.build(:user)
+    tom = FactoryGirl.build(:user, first_name: "tom")
     course = FactoryGirl.create(:course)
-    john = User.create(
-      first_name: "John",
-      last_name: "Smith",
-      email: "js@email.com",
-      course_id: course.id
-    )
-    jeff = User.create(
-      first_name: "Jeff",
-      last_name: "Sessions",
-      email: "jjj@email.com",
-      course_id: course.id
-    )
+    
+    course.users << john << tom 
```

Comment:
"has many users" test has now been refactored with the reflect_on_association method